package.json

```diff
@@ -9,7 +28,7 @@
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
     "lint:ts": "eslint 'test/**/*.ts' --max-warnings 0 --fix",
-    "typechain": "TS_NODE_TRANSPILE_ONLY=true hardhat typechain",
+    "typechain": "TS_NODE_TRANSPILE_ONLY=true yarn hardhat typechain",
```

Comment:
Why is this required? Since hardhat is a dependency it shouldn't be necessary

Reply:
getting the error below when I try to install package demo repo:
<img width="1086" alt="Screenshot 2023-07-13 at 18 05 53" src="https://github.com/5afe/safe-core-protocol/assets/16358982/3fd72a64-116b-4b26-8574-735730ab3bac">

Reply:
and if hardhat is installed globally, postinstall still fails.